app.js

```diff
@@ -1,10 +1,26 @@
 const request = require('request');
+const yargs = require('yargs');
 
 const googleApiKey = 'AIzaSyAXor0x9xSEwP-ju9o7YOdxuGX9yPl6X7E';

```

Comment:
this can be fetched from env variables like:

const googleApiKey = process.env.CONST_NAME;